docs/release-notes.md

```diff
@@ -25,6 +25,7 @@ Starting with this release, ignition-validate binaries are signed with the
 - Document that `hash` fields describe decompressed data
 - Clarify documentation of `passwordHash` fields
 - Correctly document Tang `advertisement` field as optional
+- Correctly check for noop when disabling unit's
```

Comment:
"Correctly check" suggests we weren't doing it correctly before, when in fact systemd got stricter.  Let's refocus this around working with newer systemd?

Also, "unit's" is the possessive.